### PR TITLE
Doc v2 b1

### PR DESCRIPTION
**Pull request by jtlap**

### CODE REVIEW

File: include/eve/module/core/regular/bit_flip.hpp [hunk 1]
```diff
@@ -31,7 +31,7 @@ namespace eve
 //! @addtogroup core_bitops
 //! @{
 //!   @var bit_flip
-//!   @brief flip the value the ith bit of each element.
+//!   @brief  `strict_elementwise_callable` object fliping the value the ith bit of each element.
```
Comment (jfalcou) [inline include/eve/module/core/regular/bit_flip.hpp:34]: flipping

File: include/eve/module/core/regular/bit_select.hpp [hunk 2]
```diff
@@ -42,36 +42,28 @@ namespace eve
 //!   @code
 //!   namespace eve
 //!   {
-//!     template< value T, value U > U bit_select)( T m, U x, U y )
-//!      noexcept;
+//!     template < value M, value T, value U > constexpr auto bit_select((Logical<M> m, T x, U y) noexcept;
```
Comment (jfalcou) [inline include/eve/module/core/regular/bit_select.hpp:45]: why not `bit_select(logical_value auto m, value auto x, value auto y)` like everywhere else ?

File: include/eve/module/core/regular/bit_set.hpp [hunk 3]
```diff
@@ -45,33 +45,29 @@ namespace eve
 //!   @code
 //!   namespace eve
 //!   {
-//!      template< unsigned_value T, integral_value I>
-//!      T bit_set(T x, I i) noexcept;
+//!      // Regular overload
+//!      constexpr auto bit_set(unsigned_value auto x integral_scalar_value n) noexcept;             // 1
```
Comment (jfalcou) [inline include/eve/module/core/regular/bit_set.hpp:49]: integral_scaalr_value auto

File: include/eve/module/core/regular/bit_notor.hpp [hunk 4]
```diff
@@ -49,37 +49,36 @@ namespace eve
...
 //!   namespace eve
 //!   {
-//!      template< eve::value T, eve::value Ts... >
-//!      bit_value<T, Ts...> bit_notor(T x, Ts... xs) noexcept;
+//!      // Regular overloads
+//!      constexpr auto bit_notor(value auto x, value auto ... xs)                          noexcept; // 1
+//!      constexpr auto bit_notor(kumi::non_empty_product_type auto const& tup)             noexcept; // 2
+//!
+//!      // Lanes masking
+//!      constexpr auto bit_notor[conditional_expr auto c](value auto x, value auto ... xs) noexcept; // 3
```
Comment (jfalcou) [inline include/eve/module/core/regular/bit_notor.hpp:57]: This should use the /* any of th eabove overload*/ comments liek everywhere else

**Review by jfalcou — CHANGES_REQUESTED** [review]
A lot of inconsistencies:
 - missing auto after concepts
 - you don't use the `/* any of the above overloads */` comment everywhere

File: include/eve/module/core/regular/bit_notor.hpp (hunk 4 above)
Comment (jtlap) [reply include/eve/module/core/regular/bit_notor.hpp:57]: perhaps but then, why only bit_notor ?

Comment (jtlap) [reply include/eve/module/core/regular/bit_notor.hpp:57]: moreover can we apply masks with tuple calls ?

Comment (jfalcou) [reply include/eve/module/core/regular/bit_notor.hpp:57]: I put it there but as expalined later, it is in all functions

Comment (jtlap) [reply include/eve/module/core/regular/bit_notor.hpp:57]: however done

**Pull request closed without merging**